agent/fw_drupal8.c

```diff
@@ -730,6 +785,32 @@ void nr_drupal8_enable(TSRMLS_D) {
                                     "er::getControllerFromDefinition"),
                             nr_drupal8_name_the_wt TSRMLS_CC);
 
+  /*
+   * ExceptionSubscribers handle Drupal errors and exceptions before
+   * the agent has the opportunity to capture them. Instrument several
+   * of these ExceptionSubscriber function `onException` methods in order
+   * to capture Exceptions and Errors in Drupal 9.x+
```

Comment:
Why these particular subscribers were selected?

Reply:
Removed the HttpExceptionSubscriber wrapper in favor of just the Logging and Final Exception Subscribers. Logging Subscriber should be the highest priority, since it logs an exception, but doesn't handle it. Final Subscriber is the last line of defense in case an exception has escaped all other subscribers.